.gitignore

```diff
@@ -351,5 +351,8 @@ MigrationBackup/
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
 
-# Ignore default autorest generated folder in teh root only.
-/generated
\ No newline at end of file
+# Ignores generated files from samples
+samples/**/*.yaml
```

Comment:
Do we really want to ignore yaml files?

Reply:
Yes, because they get generated as part of testing the pipeline.

Reply:
That means that we will have to force add any new samples that are yaml instead of json. Can we instead add a sub-folder convention to the output-path readmes? Instead of `.` have `.\output` or something we can then wildcard on?

Reply:
For now, this is temporary. You had asked before how I was testing changes locally. So, I figured it would be good to add what I already have to the repo. None of these files are intended to remain long-term.